### PR TITLE
API stability - phase 2

### PR DESCRIPTION
**Pull request by YusukeHosonuma**

### CODE REVIEW

File: Sources/API/Debug.swift [hunk 1]
```diff
@@ -1,51 +1,60 @@
 public class Debug {
+    /// Default format option
     public static let defaultOption = Option(indent: 4)
```
Comment (YusukeHosonuma) [inline Sources/API/Debug.swift:3]: Should I make to `private` ...?

File: Tests/API/DebugTests.swift [hunk 2]
```diff
@@ -13,6 +13,8 @@ private struct DogId {
     var rawValue: String
 }
 
+// TODO: organize test to comprehensive
```
Comment (YusukeHosonuma) [inline Tests/API/DebugTests.swift:16]: I'll do it soon, maybe... :)

File: Example/SwiftPrettyPrintExample/Source/AppDelegate.swift [hunk 3]
```diff
@@ -6,11 +6,17 @@
...
+import SwiftPrettyPrint
 import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // -------------------------
+        // Configure format-options
+        // -------------------------
+        Debug.sharedOption = Option(indent: 4)
```
Comment (po-miyasaka) [inline Example/SwiftPrettyPrintExample/Source/AppDelegate.swift]: This comment is nothing to do with this PR..
`Option` is a general word. 
so It is better that this is in `Debug`'s name space.

https://github.com/YusukeHosonuma/SwiftPrettyPrint/blob/887741e497ad1d6bc520a934cb07fb92ea82963b/Sources/API/Option.swift

**Comment by po-miyasaka** [comment]
That’s all for my comment.

File: Example/SwiftPrettyPrintExample/Source/AppDelegate.swift (hunk 3 above)
Comment (sahara-ooga) [reply Example/SwiftPrettyPrintExample/Source/AppDelegate.swift]: I have got the same impression.
`Option` could be in some module explicitly.

Comment (YusukeHosonuma) [reply Example/SwiftPrettyPrintExample/Source/AppDelegate.swift]: Thanks for feedbacks!!
I also thought to, I'll fix it.

Comment (YusukeHosonuma) [reply Example/SwiftPrettyPrintExample/Source/AppDelegate.swift]: Fixed!! 108ea4a

**Review by po-miyasaka — APPROVED** [review]
🙆‍♂️🙆‍♀️

File: README.md [hunk 4]
```diff
@@ -106,6 +106,31 @@ let dog = Dog(id: DogId(rawValue: "pochi"), price: Price(rawValue: 10.0), name:
 Debug.print(dog)
 ```
 
+## Format options
+
+You can configure format option, shared or passed by arguments:
+
+```swift
+// Global option
+Debug.sharedOption = Debug.Option(indent: 4)
```
Comment (sahara-ooga) [inline README.md:115]: Looks good to me😃

**Review by sahara-ooga — APPROVED** [review]
👍🏻

File: Sources/API/Debug.swift (hunk 1 above)
Comment (sahara-ooga) [reply Sources/API/Debug.swift:3]: I think so.

Comment (YusukeHosonuma) [reply Sources/API/Debug.swift:3]: I've been on the fence but I leave it as is.

**Comment by YusukeHosonuma** [comment]
Thanks for reviewed!!

**Pull request merged**